vaccine_feed_ingest/runners/nv/immunizenevada_org/normalize.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest.schema import schema
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+
+
+def _get_id(site: dict) -> str:
+    return f"immunizenevada_org:{site['id']}"
+
+
+def _get_address(address: str) -> schema.Address:
+    parts = address.split(",")
+    street2 = None
+    if len(parts) == 5:
+        street2 = parts[1]
+
+    result = schema.Address(
+        street1=parts[0],
+        street2=street2,
+        city=parts[-3],
+        state=parts[-2],
+        zip=parts[-1],
+    )
+    return result
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = None
+
+    phone = None
+    if "contact-phone" in site:
+        phone = site["contact-phone"]
+
+    # Filter out sites with a url of "/"
+    website = None
+    if site["url"].startswith("http"):
+        website = site["url"]
+
+    if phone or website:
+        general_contact = schema.Contact(
+            contact_type="general",
+            phone=phone,
+            website=website,
+        )
+        contacts = [general_contact]
```

Comment:
Please add separate contact links for each method - e.g., if there is a `phone` and a `website`, we should return an array of two contacts not just one.

We're chatting about if it would make sense to adjust our overall schema to be more like what you've done here, but for now let's be consistent with the other normalizers.

Reply:
To close the loop - we decided to keep it with the split contact entries for each method, and plan to build enforcement into the data schema to facilitate that.

The reason for this split is that we want to force downstream systems to handle multiple values for a single contact type (e.g., multiple general phone numbers).